setup.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Setup script."""
+
+from setuptools import find_packages
+from setuptools import setup
+
+setup(
+    name="keras-nlp",
+    description="High-level NLP libraries based on Keras",
```

Comment:
Let's sync up this description with keras-cv?
https://github.com/keras-team/keras-cv/blob/master/setup.py

Reply:
yes! done